package.json

```diff
@@ -9,9 +9,9 @@
     "format": "prettier --write .",
     "lint": "eslint packages/*/{src,types}/**",
     "test": "run-s test-serve test-build",
-    "test-serve": "jest",
+    "test-serve": "node ./scripts/setup.cjs && uvu packages/playground \"(alias|assets|cli)/__tests__/.*\\.spec\\.[jt]sx?\" -r jiti/register && node ./scripts/teardown.cjs",
```

Comment:
1) very brittle, teardown may not be called if a previous step didn't exit with 0
2) only selects a few tests instead of all, why?

Reply:
2. This PR for now only touches alias, asserts, and cli playground, so only run those for now. Other playground tests will fail because of `describe` which doesn't exist in `uvu`.